docs/development/releasing.md

```diff
@@ -147,30 +178,56 @@ For a `minor` release, e.g. `1.8.0`, use this format:
 ```
 
 ### Beta releases
-Use the same formats as the ones suggested above, but add the following note at the beginning of the changelog:
+Use the same formats as the ones suggested above, but add the following note at
+the beginning of the changelog:
 ```sh
-**Note:** This is a beta release and most likely will have have some instability and/or regressions.
+**Note:** This is a beta release and most likely will have have some
+instability and/or regressions.
 
 Please file new issues for any bugs you find in it.
 
-This release is published to [npm](https://www.npmjs.com/package/electron) under the `beta` tag and can be installed via `npm install electron@beta`.
+This release is published to [npm](https://www.npmjs.com/package/electron)
+under the `beta` tag and can be installed via `npm install electron@beta`.
 ```
 
 
 ## Edit the release draft
 
-1. Visit [the releases page] and you'll see a new draft release with placeholder release notes.
-1. Edit the release and add release notes.
-1. Uncheck the `prerelease` checkbox if you're publishing a stable release; leave it checked for beta releases.
-1. Click 'Save draft'. **Do not click 'Publish release'!**
-1. Wait for all builds to pass before proceeding.
-1. You can run `npm run release -- --validateRelease` to verify that all of the
-required files have been created for the release.
+1. Visit [the releases page] and you'll see a new draft release with placeholder
+release notes.
+2. Edit the release and add release notes.
+3. Uncheck the `prerelease` checkbox if you're publishing a stable release;
+leave it checked for beta releases.
+4. Click 'Save draft'. **Do not click 'Publish release'!**
+5. Wait for all builds to pass before proceeding.
+6. In the `release` branch, verify that the release's files have been created:
+```sh
+$ git rev-parse --abbrev-ref HEAD
+release
+$ npm run release -- --validateRelease
+```
 
+## Merge temporary branch
```

Comment:
This is only needed for pre 2.0.0 branches.  2-0-x does not use a release branch and doesn't need to be merged.